typings/webhook.d.ts

```diff
@@ -0,0 +1,20 @@
+import { MetaSys, MetaSysProps } from './meta'
+import { DefaultElements } from './defaultElements'
+
+export interface WebhookProps {
```

Comment:
You are missing the transformation property here. This has a structure like this:
```
transformation?: {
    method?: string,
    contentType?: string,
    includeContentLength?: boolean,
    body?: any
}
```
Source: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/webhooks under the section "Transforming webhook calls"

Reply:
You're right I implemented it in the last commit :). Thanks for your help